.drone.yml

```diff
@@ -81,6 +78,12 @@ pipeline:
       branch: [ refs/tags/* ]
       status: success
 
+  report-coverage:
+     image: robertstettner/drone-codecov
+     token: ${CODECOV_TOKEN}
+     files:
+       - h5c/vic/src/vic-webapp/coverage/lcov.info
```

Comment:
@jooskim AFAICT, the java tests do not (have never?) generated any coverage data. 
```
run-test:
    [mkdir] Created dir: /go/src/github.com/vmware/vic-ui/h5c/vic-service/unittest-results
    [junit] Running com.vmware.vic.test.ConfigLoaderTest
    [junit] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.077 sec
    [junit] Running com.vmware.vic.test.ContainerVmTest
    [junit] Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.255 sec
    [junit] Running com.vmware.vic.test.ModelObjectUriResolverTest
    [junit] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.077 sec
    [junit] Running com.vmware.vic.test.RootTest
    [junit] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.077 sec
    [junit] Running com.vmware.vic.test.VirtualContainerHostVmTest
    [junit] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.249 sec
    [junit] Running com.vmware.vic.test.VmQueryResultTest
    [junit] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.247 sec
```
In order to actually generate coverage data for them, something like jacoco would need to be included in your ant xml file...

Reply:
From everything I can find, this just runs the unit tests without coverage generation:
https://github.com/vmware/vic-ui/blob/master/h5c/vic-service/build-java.xml#L78